dm/dm/ctl/master/operate_source.go

```diff
@@ -32,11 +32,12 @@ import (
 // NewOperateSourceCmd creates a OperateSource command.
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> [config-file ...] [--print-sample-config]",
+		Use:   "operate-source <operate-type> [config-file ...] <-w worker> [--print-sample-config]",
```

Comment:
```suggestion
		Use:   "operate-source <operate-type> [config-file ...] [-w worker] [--print-sample-config]",
```

`<>` is used for mandatory parameters, here I think worker is optional